.github/workflows/lib-injection.yml

```diff
@@ -57,3 +57,43 @@ jobs:
           docker-registry-username: ${{ github.repository_owner }}
           docker-registry-password: ${{ secrets.GITHUB_TOKEN }}
           test-script: ./lib-injection/run-manual-lib-injection.sh
+
+  test-negative:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        variant:
+          - dd-lib-ruby-init-test-rails-conflict
```

Comment:
I'm curious -- do you know why there's no error message in https://github.com/DataDog/dd-trace-rb/actions/runs/4307107275/jobs/7511764104 ?